config/etfuturum/modcompat.cfg

```diff
@@ -0,0 +1,46 @@
+# LoTR Ores that have no raw/deepslate modded alternative will not be tagged.
+##########################################################################################################
+
+misc {
+    # Disables just the andesite, granite, and diorite full blocks, but leaves their stairs and slabs, etc enabled.
+    # Useful if you have mods like Chisel or Botania which feature these same stones but not the stairs and other variants. [default: false]
+    B:disableBaseBountifulStonesOnly=false
+
+    # Disables copper ingots and ores, but leaves the blocks and other stuff.
+    # Useful if you prefer another mod's copper, but want to use the oxidizing copper building blocks. [default: false]
+    B:disableCopperOreAndIngotOnly=false
+
+    # Adds compat for Baubles Expanded. Does nothing for standard baubles, this REQUIRES Baubles Expanded! It uses the new "wings" slot added by the expanded version. https://modrinth.com/mod/baubles-expanded
+    # When enabled, this allows the player to equip an elytra with a chestplate, by placing the elytra in a "wings" slot instead of the chestplate slot. Note that the player can only equip one elytra at a time.
+    # 0 = No compat, do not allow the elytra to be placed in a wings slot.
+    # 1 = Elytra will be placeable in a wings slot. Will enable the slot, if it isn't there.
+    # 2 = The elytra can ONLY go in the wings slots, not the chestplate slot. [range: 0 ~ 2, default: 1]
+    I:elytraBaublesExpandedCompat=1
+
+    # Deepslate ores for modded ores. Adds a set of "general" deepslate ores for common metals like "oreAluminium", "oreTin", etc, as well as explicit support for numerous mods. [default: true]
+    B:moddedDeepslateOres=true
+
+    # List of modded deepslate ores to disable. Add a ModID or ore dictionary tag. For example adding "oreTin" disables deepslate tin, and adding "SimpleOres" would disable deepslate adamantium ore. CaSe-SeNsItIvE!
+    # Each entry is separated by a new line. This only disables deepslate ores added from Et Futurum's end and will not affect deepslate ores from other mods. [default: ]
+    S:moddedDeepslateOresBlacklist <
+     >
+
+    # Raw ores for modded metals. Adds a set of "general" raw ores for common metals like "oreAluminium", "oreTin", etc. [default: true]
+    B:moddedRawOres=true
+
+    # List of modded raw ores to disable. Add a ModID or ore dictionary tag. For example adding "oreTin" disables modded raw tin, and adding "SimpleOres" would disable raw adamantium ore. CaSe-SeNsItIvE!
+    # Each entry is separated by a new line. This only disables raw ores added from Et Futurum's end and will not affect raw ores from other mods. [default: ]
+    S:moddedRawOresBlacklist <
+     >
+
+    # If Iron Chests is installed, allow Iron Shulker boxes to be crafted having all the same tiers as Iron Chests. This option does nothing if Iron Chests is not installed. [default: true]
+    B:shulkerBoxesIronChest=true
```

Comment:
sounds pretty unbalanced. but I guess even normal shulker might be depending on the recipe. we have the compressed chest for this purpose.

Reply:
I was going to gregify the recipe. Happy to discuss what that looks like or disable if you think that makes more sense.

Reply:
imo it should be around HV, but cheaper than the compressed chest, as the shulker box has only 27 slots.